common.c

```diff
@@ -81,6 +81,10 @@
 #include <netdb.h>
 #include <ifaddrs.h>
 
+#ifdef __OpenBSD__
+#define random arc4random
```

Comment:
OpenBSD gives obnoxious warnings about using random and arc4random is perfectly good for the usage in dhcp6c.